packages/markdownviewer/style/index.css

```diff
@@ -1,35 +0,0 @@
-/*-----------------------------------------------------------------------------
-| Copyright (c) Jupyter Development Team.
-| Distributed under the terms of the Modified BSD License.
-|----------------------------------------------------------------------------*/
-
-:root {
-  --jp-private-markdownviewer-padding: 16px;
-}
-
-.jp-MarkdownViewer {
-  border-top: var(--jp-border-width) solid var(--jp-border-color2);
-  outline: none;
-  overflow: hidden;
-  margin-top: -1px;
-  display: flex;
-  flex-direction: column;
-}
-
-
-.jp-MarkdownViewer .jp-Toolbar {
-    display: block;
-    height: var(--jp-toolbar-micro-height);
-    background: var(--jp-toolbar-background);
-    border-bottom: 1px solid var(--jp-toolbar-border-color);
-    box-shadow: var(--jp-toolbar-box-shadow);
-    z-index: 1;
-}
-
-.jp-MarkdownViewer .jp-RenderedMarkdown {
```

Comment:
All of this CSS is missing now. We probably have the CSS classes needed to keep it, but we will need to add it as something like `jp-Document.jp-MimeRenderer jp.RenderedMarkdown`.